invenio_oauthclient/views/client.py

```diff
@@ -126,7 +126,12 @@ def authorized(remote_app=None):
            not(current_app.debug or current_app.testing)):
             abort(403)
 
-    return current_oauthclient.handlers[remote_app]()
+    try:
+        handler = current_oauthclient.handlers[remote_app]()
+    except RuntimeError:
```

Comment:
Based on the stacktrace you included, shouldn't this exception be more specific (e.g. OAuthException). Also, I'd like to understand why the exception happens - is this only for CERN or is it also for e.g. ORCID? Catching the error at this level could potentially mask quite a lot of errors which would then never be reported, hence I'd prefer to be a bit careful and understand exactly why.